### ISSUE TITLE
Change label from <none> to <manual Input> in Browse dialogs

### ISSUE THREAD

**Issue by b-jesch** [xbmc/xbmc#24547]
## Roadmap or todo item
**Detailed Description**

Please change the label from `<none>` to `<manual Input>` in BrowseDialogs like File Manager or "add source" Dialogs. 

### Context

If you want to access paths in the file system or in the network and you have to enter these paths manually, the `<none>` label is misleading. However, this is the only way to enter them manually. I have already seen many posts in the forums where users have failed to make such an entry because the `<none>` label is not appropriate. This applies in particular to paths in Android systems, but also affects other systems.

Therefore a label change to `manual Input` has most benefit and is more understandable for the users.

### Possible Implementation

Maybe only a change in the language ressource files number 231 is needed.

### Additional context, screenshots or links
Here are some relevant links or screenshots

See before, after:

![Bildschirmfoto von 2024-01-20 09-59-51](https://github.com/xbmc/xbmc/assets/10152384/489df807-cce9-420e-a848-910c75ce3749)

![Bildschirmfoto von 2024-01-20 10-11-52](https://github.com/xbmc/xbmc/assets/10152384/9e730e4e-414e-4612-941c-e8c8978dca47)

*Note: Team Kodi is considering this item, but we are not making any promises if and when it will be included.*

**Comment by malard** [comment]
I don't disagree that <None> is not user friendly to understand, the question is, is 'Anything' i.e. should we just leave the window blank and is it then more clear that nothing has been put in?

Also should the window be called "Add file sources" given that you can add more than one, in fact should it be "Add media source" (to match the label below)

I think this is a bigger 'issue' in that none of the naming matches

**Comment by b-jesch** [comment]
The label of the dialog window is not as problematic, the `<None>` irritates some people and avoid them to click/navigate onto this input field. They often mix up `<None>` with `<Nothing>`.

> should we just leave the window blank and is it then more clear that nothing has been put in?

I don't think so. `<manual input>` should be generic enough.

**Comment by malard** [comment]
I am sorry but I don't agree that <manual input> actually solves anything. In fact that to me reads worse. Do you have any examples of other applications and how they handle this sort of modal?

**Comment by b-jesch** [comment]
No, but I have some forums threads where people are suspicious about the naming of the pre-assignment of this input field. Example? See: https://www.kodinerds.net/thread/77503-magicsee-n5max-vs-bqeel-s1max-vs-nokia-8010/?postID=741265#post741265
Sorry for german, use the builtin translator of your browser.

**Comment by jjd-uk** [comment]
The `Enter the paths or browse for the media location` is supposed to prompt the user what to do. So if that is not obvious enough it is that label that should be adjusted.

**Comment by jjd-uk** [comment]
I think it's generic as I believe that string maybe used in other places. I'm not sure as I haven't checked, but I think it maybe used in the smartplaylist editor when there are no rules.

**Comment by jjd-uk** [comment]
PR has been raised seeking to address this. Been on to do list to look at since my comment last year however other priorities mean I've only now got to looking at it.

**Comment by jjd-uk** [comment]
Closing this as it's not a bug but by design. However will be resolved is PR is accepted.
